integration/addition_gids_test.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 func TestAdditionalGids(t *testing.T) {
-	testPodLogDir, err := os.MkdirTemp("/tmp", "additional-gids")
+	testPodLogDir, err := os.MkdirTemp("", "additional-gids")
```

Comment:
better to use `t.TempDir()`

https://pkg.go.dev/testing#T.TempDir

Reply:
Updated!